README.md

```diff
@@ -269,6 +270,26 @@ spectator.query(byLabel('By label'));
 spectator.query(byText('By text'));
 ```
 
+
+#### Mocking components
+
+If you need to mock components, you can use the [ng-mocks](https://github.com/ike18t/ng-mocks) library. Instead of using `CUSTOM_ELEMENTS_SCHEMA` which might hide some issues and won't help you to set inputs, outputs, etc, `ng-mocks` will auto mock the inputs, outputs, etc for you.
+
+Example:
+
+```ts
+import { createHostComponentFactory } from '@netbasal/spectator';
```

Comment:
@ngneat

Reply:
`createHostFactory`

Reply:
Woups :zipper_mouth_face: took that from an old comment in the thread about ng-mocks :man_facepalming: